packages/flutter/lib/src/cupertino/page_scaffold.dart

```diff
@@ -75,11 +75,11 @@ class CupertinoPageScaffold extends StatefulWidget {
 }
 
 class _CupertinoPageScaffoldState extends State<CupertinoPageScaffold> {
-  final ScrollController _primaryScrollController = ScrollController();
 
   void _handleStatusBarTap() {
+    final ScrollController? _primaryScrollController = PrimaryScrollController.of(context);
```

Comment:
Is there a good place where we could document on this class that it uses the PrimaryScrollController from the environment?

Reply:
Added